### PR TITLE
Move `PictureInPictureConfig` to `PlayerViewConfig`

### PR DESCRIPTION
**Pull request by rolandkakonyi**
## Description

With the introduction of `PlayerViewConfig` it makes sense to move `PictureInPictureConfig` into it, as it is on the iOS SDK for example [here](https://cdn.bitmovin.com/player/ios/3/docs/Classes/PlayerViewConfig.html#/c:@M@BitmovinPlayerCore@objc(cs)BMPPlayerViewConfig(py)pictureInPictureConfig).

## Changes

Adapted parsing with introduction of internal types.

!!!IMPORTANT!!!
This is not an API breaking change because the `PictureInPictureConfig` was not yet released!

+1 made `PlayerViewConfig.uiConfig` optional on the TypeScript side to allow simpler configuration if one wants to only set `pictureInPictureConfig`.
+2 removing padding on Android when entering PiP to not show borders on the PiP view.

## Checklist
- [ ] 🗒 `CHANGELOG` entry - not applicable

### CODE REVIEW

File: example/src/screens/BasicPictureInPicture.tsx [hunk 1]
```diff
@@ -115,12 +117,20 @@ export default function BasicPictureInPicture({
...
-    <SafeAreaView edges={['bottom', 'left', 'right']} style={styles.container}>
+    <SafeAreaView
+      edges={['bottom', 'left', 'right']}
+      style={
+        // On Android, we need to remove the padding from the container when in PiP mode.
+        Platform.OS === 'android' && isInPictureInPicture
+          ? [styles.container, { padding: 0 }]
+          : styles.container
+      }
+    >
```
Comment (rolandkakonyi) [inline example/src/screens/BasicPictureInPicture.tsx:128]: This is a sneak to remove padding when in PiP view as it shows up on Android

File: android/src/main/java/com/bitmovin/player/reactnative/RNPlayerView.kt [hunk 2]
```diff
@@ -98,6 +98,10 @@ class RNPlayerView(val context: ReactApplicationContext) :
     LifecycleEventListener,
     View.OnLayoutChangeListener,
     RNPictureInPictureDelegate {
+    data class RNPlayerViewConfig(
+        val playerViewConfig: PlayerViewConfig?,
+        val pictureInPictureConfig: RNPictureInPictureHandler.PictureInPictureConfig?,
+    )
```
Comment (strangesource) [inline android/src/main/java/com/bitmovin/player/reactnative/RNPlayerView.kt:104]: NIT: I would prefer to not have that as a nested class in the RNPlayerView. I don't think we have this particular situation yet, I would suggest o move it to a separate file adjacent to the `RNPlayerView` or to the bottom on `RNPlayerView.kt`.

Would also be nice to have an explanation around why we have this wrapper. Naming wise I would prefer `RNPlayerViewConfigWrapper`.

**Review by strangesource — COMMENTED** [review]
Nice improvements! 
Just one minor remark around the Android native part.

File: android/src/main/java/com/bitmovin/player/reactnative/RNPlayerView.kt (hunk 2 above)
Comment (rolandkakonyi) [reply android/src/main/java/com/bitmovin/player/reactnative/RNPlayerView.kt]: fixed in e233e73

**Pull request merged**